### ISSUE TITLE
Expose applications available in flathub-beta

### ISSUE THREAD

**Issue by barthalion** [flathub/backend#22]
Currently flathub-beta is pretty much non-existent to regular users. We don't mention it anywhere on the website (for valid reasons), however it would be nice idea to show a "this app has beta program" sign on applications both in stable and beta repos, similarly to what Play Store does.

![image](https://user-images.githubusercontent.com/380403/102872040-7adf1200-443f-11eb-8594-425f7841f5e4.png)

This means we need a `/beta` endpoint, either returning full appstream or just `true`.

**Comment by tchx84** [comment]
Following the "this app has beta program"" idea, maybe simply something we could display on the existing app page? e.g. an expander that displays the same installation instructions but for installing the beta version instead.

I don't think Flatpak has the same mechanisms that Steam client does to opt-in and out from betas (or any branch in particular), so the approach mentioned above could be a less expensive.

**Comment by barthalion** [comment]
Yeah, basically "if /beta/appid returns 200, show instructions for beta" on the frontend side.

**Comment by TheEvilSkeleton** [comment]
I like this idea. People sometimes open issues and ask why some programs are unavailable in https://flathub.org/. Most Chromium-based browsers are only in the beta branch.

**Comment by BEEDELLROKEJULIANLOCKHART** [comment]
[If the beta repository is exposed, great maintenance shall need to be completed to ensure that old software is not being provided.](http://discourse.flathub.org/t/1710)

**Comment by razzeee** [comment]
> Yeah, basically "if /beta/appid returns 200, show instructions for beta" on the frontend side.

This would mean, that apps, that are only beta, wouldn't show up on flathub at all.

My very basic draft in https://github.com/flathub/backend/pull/79 also adds the appstream data for beta only apps to redis. But I'm not really happy with that approach either (yet).

**Comment by razzeee** [comment]
https://user-images.githubusercontent.com/5943908/144319160-b588eb72-0536-48ab-a497-a169dcbd1d1e.mp4

**Comment by TheEvilSkeleton** [comment]
Spectacular!
